cli_reference/basic_cli_operations.adoc

```diff
@@ -180,28 +272,36 @@ Use the `osc rollback` command to revert part of an application back to a previo
 |Instead of performing the rollback, describe what the rollback will look like in human-readable form.
 
 .^|`-o, --output`
-|Instead of performing the rollback, print the updated deployment configuration in the specified format (json\|yaml\|template\|templatefile).
+|Instead of performing the rollback, print the updated deployment configuration in the specified format: `json`\|`yaml`\|`template`\|`templatefile`.
 
 .^|`-t, --template`
-|Template string or path to template file to use when -o=template or -o=templatefile.
+|Template string or path to template file to use when ``-o=template` or `-o=templatefile`.
 |===
```

Comment:
There's no reason to have the command options spread out across two lines, they should be on a single line. Unless it's just GitHub formatting, you should increase the width of the first column to ensure that.